src/js/entities-service/entities/actions.js

```diff
@@ -250,7 +249,12 @@ const _Model = BaseModel.extend({
 
     return !!size(programAction.get('allowed_uploads'));
   },
-  removeAttachment(resource) {
+  addFileRelationship(resource) {
```

Comment:
I think we can remove the "Relationship" from the name  The other instances like this don't have it.. `addTag` `removeTag` etc

Reply:
Makes sense, updated to that 👍